<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemTest</class>
 <widget class="QWidget" name="SystemTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel
{
	color: white;
	border: 2px groove, gray;
    border-color: gray;
	font: 18px, &quot;微软雅黑&quot;;
}

QWidget
{
	background-color: black;
}

QPushButton
{
	font: 22px, &quot;微软雅黑&quot;;
    color: white;
	background-color: black;
	border:2px solid white;
}</string>
  </property>
  <widget class="QPushButton" name="btnConfirm">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>420</y>
     <width>85</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	font: 18px, &quot;微软雅黑&quot;;
    color: black;
	background-color: rgb(113, 113, 198);
	border:2px solid white; 
    border-right-color: rgb(113, 113, 198); 
    border-bottom-color: rgb(113, 113, 198);
}

QPushButton:disabled
{
	font: 18px, &quot;微软雅黑&quot;;
    color: white;
	background-color: rgb(113, 113, 198);
	border:2px solid white; 
    border-right-color: rgb(113, 113, 198); 
    border-bottom-color: rgb(113, 113, 198);
}

QPushButton:pressed
{
	font: 18px, &quot;微软雅黑&quot;;
    color: white;
	background-color: rgb(0, 0, 255); 
	border:2px solid white; 
	border-left-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 0, 
                        stop: 0 gray, stop: 1 black); 			border-top-color: qlineargradient(x1: 0, y1: 0, x2: 0, 										y2: 1, stop: 0 gray, stop: 1 black);
  border-right-color: white; border-bottom-color: white;
}</string>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>490</y>
     <width>85</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	font: 18px, &quot;微软雅黑&quot;;
    color: black;
	background-color: rgb(113, 113, 198);
	border:2px solid white; 
    border-right-color: rgb(113, 113, 198); 
    border-bottom-color: rgb(113, 113, 198);
}

QPushButton:disabled
{
	font: 18px, &quot;微软雅黑&quot;;
    color: white;
	background-color: rgb(113, 113, 198);
	border:2px solid white; 
    border-right-color: rgb(113, 113, 198); 
    border-bottom-color: rgb(113, 113, 198);
}

QPushButton:pressed
{
	font: 18px, &quot;微软雅黑&quot;;
    color: white;
	background-color: rgb(0, 0, 255); 
	border:2px solid white; 
	border-left-color: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 0, 
                        stop: 0 gray, stop: 1 black); 			border-top-color: qlineargradient(x1: 0, y1: 0, x2: 0, 										y2: 1, stop: 0 gray, stop: 1 black);
  border-right-color: white; border-bottom-color: white;
}</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBrakeTest">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>591</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	font: 22px, &quot;微软雅黑&quot;;
    color: white;
	background-color: black;
	border:2px solid white;
	border-radius: 8px;
}</string>
   </property>
   <property name="text">
    <string>制动系统自检</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBrakeResistorTest">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>591</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	font: 22px, &quot;微软雅黑&quot;;
    color: white;
	background-color: black;
	border:2px solid white;
	border-radius: 8px;
}</string>
   </property>
   <property name="text">
    <string>牵引系统测试</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSivTest">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>591</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	font: 22px, &quot;微软雅黑&quot;;
    color: white;
	background-color: black;
	border:2px solid white;
	border-radius: 8px;
}</string>
   </property>
   <property name="text">
    <string>辅助系统测试</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCompressorTest">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>280</y>
     <width>591</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	font: 22px, &quot;微软雅黑&quot;;
    color: white;
	background-color: black;
	border:2px solid white;
	border-radius: 8px;
}</string>
   </property>
   <property name="text">
    <string>空压机测试</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnHandShank">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>360</y>
     <width>591</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	font: 22px, &quot;微软雅黑&quot;;
    color: white;
	background-color: black;
	border:2px solid white;
	border-radius: 8px;
}</string>
   </property>
   <property name="text">
    <string>司控器测试</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAcceleratedSpeed">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>440</y>
     <width>591</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	font: 22px, &quot;微软雅黑&quot;;
    color: white;
	background-color: black;
	border:2px solid white;
	border-radius: 8px;
}</string>
   </property>
   <property name="text">
    <string>加减速度测试</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLcuTest">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>520</y>
     <width>591</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	font: 22px, &quot;微软雅黑&quot;;
    color: white;
	background-color: black;
	border:2px solid white;
	border-radius: 8px;
}</string>
   </property>
   <property name="text">
    <string>LCU切换测试</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
